src/main/java/nerdhub/textilelib/event/client/render/hud/DebugHudTextCallback.java

```diff
@@ -0,0 +1,27 @@
+package nerdhub.textilelib.event.client.render.hud;
+
+import java.util.List;
+
+import net.minecraft.client.gui.hud.DebugHud;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+
+@Environment(EnvType.CLIENT)
+public interface DebugHudTextCallback {
+
+    Event<DebugHudTextCallback> EVENT = EventFactory.createArrayBacked(DebugHudTextCallback.class, listeners -> (List<String> list, Side side) -> {
+        for (DebugHudTextCallback callback : listeners) {
+            callback.getText(list, side);
+        }
+    });
```

Comment:
this should be cancelable, unless we have a separate event for that

Reply:
DrawHud can be cancelled, but that should cancel the entire hud and just not one side

Reply:
I agree